roles/deploy/hooks/build-after.yml

```diff
@@ -21,3 +21,9 @@
   composer:
     no_scripts: yes
     working_dir: "{{ deploy_helper.new_release_path }}"
+
+- name: Gzip static assets
+  shell: "for i in `find . | grep -E \"\\.css$|\\.js$|\\.txt$|\\.html$|\\.xml$|\\.svg$|\\.ttf$|\\.otf$|\\.eot$|\\.woff$|\\.woff2$\"`; do gzip --best --keep --force \"$i\"; done"
```

Comment:
I feel like there must be better script for this, specifically the first part. 
These links might help https://stackoverflow.com/a/9067042/5106821 , and https://unix.stackexchange.com/a/47987/210518 . Also, does it make sense to use `--force` for gzip?